plugins/scaffolder/src/api.ts

```diff
@@ -214,6 +214,7 @@ export class ScaffolderClient implements ScaffolderApi {
                 subscriber.error(ex);
               }
             }
+            eventSource.close();
```

Comment:
I had a question about the line below this `subscriber.complete` and what it does. I don't see a `complete` function defined anywhere, my understanding is that https://github.com/backstage/backstage/blob/master/plugins/scaffolder/src/components/hooks/useEventStream.ts#L169 this is where we define the subscriber and I only see a `next` and `error` function there. Therefore do we need the `subscriber.complete()` below this?

Reply:
The complete is for the Observable that we return. We should definitively complete it when we are done with it.